activesupport/lib/active_support/deprecation/behaviors.rb

```diff
@@ -1,6 +1,8 @@
 require "active_support/notifications"
 
 module ActiveSupport
+
+  # Raised when <tt>ActiveSupport::Deprecation::Behavior#behavior</tt> is set with <tt>:raise</tt>
```

Comment:
I think this doc doesn't say much. Can we improve it?
Like, why if I have that set to `:raise` it would raise?